src/app/components/wrapper-components/part-creation/part-creation.component.ts

```diff
@@ -476,12 +477,9 @@ export class PartCreationComponent implements OnInit, OnDestroy {
             this.connectedUserService.sendPresenceToken();
         }, PartCreationComponent.TOKEN_INTERVAL));
         const userId: string = this.connectedUserService.user.get().id;
-        this.selfSubscription = this.userService.observeUser(userId, (userOpt: MGPOptional<User>) => {
-            assert(userOpt.isPresent(), 'connected user should exist');
-            const user: User = userOpt.get();
-            if (user.lastUpdateTime != null) {
-                this.lastToken = user.lastUpdateTime as Timestamp;
-            }
+        this.selfSubscription = this.userService.observeUserOnServer(userId, (user: MGPOptional<User>) => {
+            assert(user.isPresent(), 'connected user should exist');
+            this.lastToken = user.get().lastUpdateTime as Timestamp;
```

Comment:
un petit assert lastUpdateTime is not nullable ?

Reply:
ajouté un getNonNullable